win32ss/user/ntuser/scrollbar.c

```diff
@@ -1310,17 +1306,24 @@ NtUserEnableScrollBar(
 // Done in user32:
 //   SCROLL_RefreshScrollBar(hwnd, nBar, TRUE, TRUE);
 
-   RETURN(Chg);
-   if (OrigArrows == wArrows) RETURN(FALSE);
-   RETURN(TRUE);
+   Ret = Chg;
+   goto Cleanup; // FIXME
```

Comment:
Yes OK this keeps with the original code.

Reply:
Shouldn't we strip the dead/disabled code below maybe?
Since you are now referenced in the first hop by git blame for those code-parts, be prepared for stupid questions in the future, what it is intended for. :D
I would vote for tabula rasa.